tests/integration/test_zosmf.py

```diff
@@ -2,16 +2,16 @@
 from decouple import config
```

Comment:
@richardgreg Please remove this import which is now unused.

Also, all the other integration test files need to have the same changes made in their setUp stage as this file (replacing the load of `decouple.config` and usage of `connection_dict` with the new ProfileManager class).

After updating all the integration tests, you can comment out the `ZOWE_TEST_PROFILE` in your .env file so the tests won't be able to find a v1 profile, and verify that all the tests pass using your v2 configuration (the zowe.config.json that we set up).